tests/integration/test_disks_app_other_disk_types/config.xml

```diff
@@ -0,0 +1,31 @@
+<clickhouse>
+    <path>/var/lib/clickhouse/</path>
+    <logger>
+        <clickhouse-disks>/clickhouse-disks.log</clickhouse-disks>
+    </logger>
+    <storage_configuration>
+        <disks>
+            <plainRewritableDisk>
+                <type>object_storage</type>
+                <object_storage_type>s3</object_storage_type>
+                <metadata_type>plain_rewritable</metadata_type>
+                <endpoint>http://minio1:9001/root/data/</endpoint>
+                <endpoint_subpath>rewr_xd</endpoint_subpath>
+                <access_key_id>minio</access_key_id>
+                <secret_access_key>minio123</secret_access_key>
+            </plainRewritableDisk>
+        </disks>
+        <policies>
+            <plain_rewritable>
+                <volumes>
+                    <main><disk>plainRewritableDisk</disk></main>
+                </volumes>
+            </plain_rewritable>
+        </policies>
+    </storage_configuration>
+
+    <zookeeper>
+        <availability_zone_autodetect>true</availability_zone_autodetect>
+        <prefer_local_availability_zone>1</prefer_local_availability_zone>
+    </zookeeper>
```

Comment:
```suggestion
```

Reply:
It's used in the private repo, this probably broke the test there, I"ll check.

Reply:
Ok, it didn't make the test fail but made it ineffective. These settings are what triggered the `Placement info has not been loaded`.